### PR TITLE
Rename "system-administrator" in genesis, bootstrap files

### PR DESCRIPTION
**Pull request by Ebazhanov**
link: https://github.com/confio/tgrade/issues/328

Rename `SystemAdminAddress` to `bootstrap-account`: 
- [x] in `genesis` file
- [x] in `bootstrap` file
- [x] in `engagement` file
[....](https://github.com/confio/tgrade/pull/360/files)

### CODE REVIEW

**Comment by alpe** [comment]
Very nice. Can you grep for `systemadmin` ? There are still some places in tests and scripts not renamed

**Comment by Ebazhanov** [comment]
> Very nice. Can you grep for `systemadmin` ? There are still some places in tests and scripts not renamed

forgot 👍 , checking

File: x/poe/contract/tg4_engagement.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func NewEngagementContractAdapter(contractAddr sdk.AccAddress, twasmKeeper types
 	}
 }
 
-func (a EngagementContractAdapter) UpdateAdmin(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error {
+func (a EngagementContractAdapter) UpdateBootstrapAccountAddress(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error {
```
Comment (alpe) [inline x/poe/contract/tg4_engagement.go:80]: this is not correct. We still have an "admin" role within the PoE contract. Please revert this change

File: x/poe/contract/tg4_engagement_integration_test.go [hunk 2]
```diff
@@ -21,18 +21,18 @@ var tg4Engagement []byte
 
 func TestEngagementUpdateAdmin(t *testing.T) {
 	ctx, example := keeper.CreateDefaultTestInput(t)
-	var systemAdminAddr sdk.AccAddress = rand.Bytes(address.Len)
+	var bootstarpAccountAddr sdk.AccAddress = rand.Bytes(address.Len)
```
Comment (alpe) [inline x/poe/contract/tg4_engagement_integration_test.go:24]: typo
```suggestion
	var bootstrapAccountAddr sdk.AccAddress = rand.Bytes(address.Len)
```

File: x/poe/keeper/contracts.go [hunk 3]
```diff
@@ -51,7 +51,7 @@ func (k Keeper) StakeContract(ctx sdk.Context) StakeContract {
 }
 
 type EngagementContract interface {
-	UpdateAdmin(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error
+	UpdateBootstrapAccountAddress(ctx sdk.Context, newAdmin, sender sdk.AccAddress) error
```
Comment (alpe) [inline x/poe/keeper/contracts.go:54]: As above, please revert the renaming for this interface and other implementations

File: contrib/local/setup_tgrade.sh [hunk 4]
```diff
@@ -40,16 +40,16 @@ for addr in "$@"; do
...
 content=$(cat "$HOME"/.tgrade/config/genesis.json | jq  ".app_state.poe.seed_contracts.engagement |= . + [{\"address\":\"$(echo "$PASSWORD" | tgrade keys show -a validator)\",\"points\":\"100\"}]")
 # set oversight community
-content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.oversightCommunityMembers |= . + [\"$(echo "$PASSWORD" | tgrade keys show -a systemadmin)\"]")
+content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.oversightCommunityMembers |= . + [\"$(echo "$PASSWORD" | tgrade keys show -a bootstrap-account)\"]")
 # set arbiter
-content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.arbiterPoolMembers |= . + [\"$(echo "$PASSWORD" | tgrade keys show -a systemadmin)\"]")
+content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.arbiterPoolMembers |= . + [\"$(echo "$PASSWORD" | tgrade keys show -a bootstrap-account)\"]")
 # set system admin
-content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.system_admin_address |= \"$(echo "$PASSWORD" | tgrade keys show -a systemadmin)\"")
+content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.system_admin_address |= \"$(echo "$PASSWORD" | tgrade keys show -a bootstrap-account)\"")
```
Comment (alpe) [inline contrib/local/setup_tgrade.sh:52]:
```suggestion
content=$(echo "$content" | jq  ".app_state.poe.seed_contracts.bootstrap_account_address |= \"$(echo "$PASSWORD" | tgrade keys show -a bootstrap-account)\"")
```

File: x/poe/contract/tg4_engagement.go (hunk 1 above)
Comment (Ebazhanov) [reply x/poe/contract/tg4_engagement.go]: updated

File: contrib/local/setup_tgrade.sh (hunk 4 above)
Comment (Ebazhanov) [reply contrib/local/setup_tgrade.sh]: updated https://github.com/confio/tgrade/pull/360/commits/b1f15e1302fddc9a3431ff6a4b68e8d3dfddb07e

**Review by alpe — APPROVED** [review]
Thanks for the update! Looks great now 🥇
